web/src/containers/admin/CreateAccount.test.js

```diff
@@ -0,0 +1,25 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { plain as CreateAccount } from './CreateAccount';
+
+const testProps = {
+  createUser: () => {},
+  error: false,
+  roles: [
+    { id: 1, name: "admin", isActive: true },
+    { id: 2, name: "state coordinator", isActive: true },
+  ],
+  working: false
+};
+
+const setup = (props = {}) => {
+  return shallow(<CreateAccount {...testProps} {...props} />);
+};
+
+describe('<CreateAccount />', () => {
+  it('renders correctly', () => {
+    const component = setup();
+    expect(component).toMatchSnapshot();
+  });
+});
```

Comment:
I didn't want to put too much effort into writing a test here, since this component will change drastically when moving forward with Okta stories, and could possibly go away entirely.

Reply:
good idea